src/Utils/Hexagons.svelte

```diff
@@ -344,83 +386,136 @@ const sleep = (milliseconds) => {
-                    <text id={`text-${hexagon.id}`} x={hexagon.x - 6} y={hexagon.y}>{hexagon.id}</text>
-                </g>
-        {/each}
-    </svg>
-    {#if test_ui}
-        {#each hexagons as actuator}
-            <button on:click={buildCommandBlocks(actuator.id)}> Button {actuator.id}</button>
-        {/each}
-        <p>Test: {$block0_31}, {$block32_63}, {$block64_95}, {$block96_127}</p>
-
+<div class="full-container" style={setBackground}>
+    {#if arraySize == "normal"}
+        <div class="col-5">
+            <select bind:value={arrayType}>
+                {#each hexagonTypes as type}
+                    <option value={type.id}>
+                        {type.text}
+                    </option>
+                {/each}
+            </select>
+            <br/>
+            <button on:click={() => moveable.request("rotatable",{deltaRotate: -(rotation)}, true)}>Rotate {rotation}&#730 &#8634</button>    
+            <button on:click={() => moveable.request("rotatable",{deltaRotate: +(rotation)}, true)}>Rotate {rotation}&#730 &#8635</button>    
+            <input style="width: 50%" bind:value={rotation}/> &deg
+            <br/>
+            <button on:click={() => moveable.request("rotatable",{rotate: initialRotation}, true)}>Reset</button>
+        </div>
     {/if}
+
+    <div class={className}>
+        <svg  version="1.0" xmlns="http://www.w3.org/2000/svg" bind:this={target} style={setStyle}
+            width={`${viewBox.x}px`}
+            height={`${viewBox.y}px`}
+            viewBox={`0 0 ${viewBox.x} ${viewBox.y}`}
+            preserveAspectRatio="xMidYMid meet"
+            on:mousedown={e => handleTouchStart(e)}
+            on:mousemove={e => handleTouchMove(e)}
+            on:mouseup={e => handleTouchEnd(e)}    
+            >
+            {#each drawableHexagons as hexagon}
+                <g key={`g-${hexagon.id}`}>
+                    <polygon
+                        id={`hex-${hexagon.id}`}
+                        key={`hex-${hexagon.x}-${hexagon.y}`}
+                        class="activatable"
+                        fill="none"
+                        stroke={hexagon.color}
+                        stroke-width={strokeWidth}
+                        strokeLinejoin="miter"
+                        transform={`translate(${hexagon.x - (hexagon.width / 2)} ${hexagon.y - (hexagon.height / 2)})`}
+                        points={hexagon.points} />
+                        {#if arraySize == "normal"}
+                            <text id={`text-${hexagon.id}`} x={hexagon.x - 6} y={hexagon.y}>{hexagon.id}</text>
+                        {/if}
+                    </g>
+            {/each}
+        </svg>
+        {#if test_ui}
+            {#each hexagonsLayout as actuator}
+                <button on:click={buildCommandBlocks(actuator.id)}> Button {actuator.id}</button>
+            {/each}
+            <p>Test: {$block0_31}, {$block32_63}, {$block64_95}, {$block96_127}</p>
```

Comment:
Do the tests still work?

Reply:
No. I need to go back and fix them. I was waiting till I fixed multitouch in case that changed how the commands were built.